Interim docs for Spring Session for GF

jomartin-999 opened the pull request.

jomartin-999 committed:
Interim docs for Spring Session for GF

kohlmu-pivotal commented on documentation/release-notes.md:
```diff
@@ -0,0 +1,28 @@
+
+### 1.1.1
+
+* Update Spring Session 2.6 to Spring Session 2.6.14
```
2.6.14 -> 2.6.3

jomartin-999 replied:
Done.

kohlmu-pivotal commented on documentation/release-notes.md:
```diff
@@ -0,0 +1,28 @@
+### 1.1.1
+
+* Update Spring Session 2.6 to Spring Session 2.6.14
+* Update Spring Session 2.7 to Spring Session 2.7.10
```
2.7.10 - > 2.7.1

jomartin-999 replied:
Done

kohlmu-pivotal commented on documentation/release-notes.md:
```diff
@@ -0,0 +1,28 @@
+
+* Update Spring Session 2.6 to Spring Session 2.6.14
+* Update Spring Session 2.7 to Spring Session 2.7.10
+* Update Spring Session 3.0 to Spring Session 3.0.5
```
3.0.5 -> 3.0.1

kohlmu-pivotal commented on documentation/release-notes.md:
```diff
@@ -0,0 +1,28 @@
+* Update Spring Session 2.7 to Spring Session 2.7.10
+* Update Spring Session 3.0 to Spring Session 3.0.5
+* Update all dependencies to latest patch version
+* Fixed an issue with querying a Region whose name contains a hyphen (`-`). 
```
This is only applicable to Spring Data

jomartin-999 replied:
Done

jomartin-999 committed:
Updated release notes with correct version

Updated release notes with correct versions
Removed "Fixed issue" as it does not apply.
